src/app/cli/build.rs

```diff
@@ -18,5 +18,8 @@ fn main() -> Result<(), Box<dyn Error>> {
         .all_rustc()
         .all_cargo()
         .emit()?;
+
+    println!("cargo:rerun-if-changed=migrations");
```

Comment:
?

Reply:
Updated this piece to add descriptions of what was going on

Done